smartsim/builders/ensemble.py

```diff
@@ -249,7 +311,14 @@ def permutation_strategy(
         """Set the permutation strategy
 
         :param value: the permutation strategy
+        :raises TypeError: if permutation_strategy is not str or
+        PermutationStrategyType
         """
+        if value:
+            if not callable(value):
+                raise TypeError(
+                    "permutation_strategy argument was not of type str or PermutationStrategyType"
+                )
```

Comment:
Similar style nit here:
```suggestion
                raise TypeError(
                    "permutation_strategy argument was not of type str or "
                    "PermutationStrategyType"
                )
```

Reply:
changed